Docker images not building deb packages

when building the image I keep getting this errors

```
15-Jun-2016 16:01:56	can't load package: package github.com/QubitProducts/bamboo: 
15-Jun-2016 16:01:56	bamboo.go:1:1: expected 'package', found 'IDENT' main
15-Jun-2016 16:01:56	The command '/bin/sh -c go get github.com/tools/godep &&     go get -t github.com/smartystreets/goconvey &&     go build' returned a non-zero code: 1
```
Note: gem requires ruby installed on the docker containers otherwise it won't be  able to install fpm since golang image doesn't have ruby.

Let me try to reproduce this, will get back to you later today.


> On 15 Jun 2016, at 07:20, Rhommel Lamas <notifications@github.com> wrote:
> 
> when building the image I keep getting this errors
> 
> 15-Jun-2016 16:01:56    can't load package: package github.com/QubitProducts/bamboo: 
> 15-Jun-2016 16:01:56    bamboo.go:1:1: expected 'package', found 'IDENT' main
> 15-Jun-2016 16:01:56    The command '/bin/sh -c go get github.com/tools/godep &&     go get -t github.com/smartystreets/goconvey &&     go build' returned a non-zero code: 1
> Note: gem requires ruby installed on the docker containers otherwise it won't be able to install fpm since golang image doesn't have ruby.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

--- a/Dockerfile-deb
+++ b/Dockerfile-deb
@@ -3,7 +3,7 @@ FROM golang:1.6.2
 ENV DEBIAN_FRONTEND noninteractive
 
 RUN apt-get update -yqq
-RUN apt-get install -yqq software-properties-common curl git mercurial ruby-dev gcc make
+RUN apt-get install -yqq software-properties-common curl git mercurial ruby-dev gcc make rubygems
 RUN gem install fpm
 
 ADD . /go/src/github.com/QubitProducts/bamboo

--- a/builder/build.sh
+++ b/builder/build.sh
@@ -2,7 +2,7 @@
 set -e
 set -u
 name=bamboo
-version=${_BAMBOO_VERSION:-"1.0.0"}
+version=${_BAMBOO_VERSION:-"0.2.20"}
 description="Bamboo is a DNS based HAProxy auto configuration and auto service discovery for Mesos Marathon."
 url="https://github.com/QuBitProducts/bamboo"
 arch="all"

